website/docs/docs/deploy/cloud-ci-job.md

```diff
@@ -51,19 +60,21 @@ The green checkmark means the dbt builds and tests were successful. Clicking the
 The green checkmark means the dbt builds and tests were successful. Clicking on the dbt Cloud section navigates you to the relevant CI run in dbt Cloud.
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Enabling-CI/ADO CI Check.png" title="Azure DevOps pull request"/>
 
-## Configuring a dbt Cloud CI job
+## Configuring a Slim CI job
 
-Setting up a CI job is very similiar to setting up a normal production job that runs on a schedule; however, a CI job has some notable differences.
+Slim CI offers an alternative in dbt Cloud for running and testing all models in your project. When you're only changing a few handful of models, Slim CI can decrease the time it takes by running and testing only modified models. Slim CI enables you to focus on testing changed models and reduces your overall cost. 
```

Comment:
Slim CI isn't unique to dbt Cloud, it just is _way_ easier to do it in Cloud because we manage all the artifacts etc for you

>Slim CI offers an alternative to running and testing all models in your project, even when you're only changed a couple of things. Slim CI can decrease the time it takes by running and testing only modified models, which can also reduce unnecessary resource usage on your warehouse/data platform.